extensions/theme-defaults/themes/dark_vs.json

```diff
@@ -12,8 +12,8 @@
 		"activityBarBadge.background": "#007ACC",
 		"sideBarTitle.foreground": "#BBBBBB",
 		"input.placeholderForeground": "#A6A6A6",
-		"menu.background": "#252526",
-		"menu.foreground": "#CCCCCC",
+		"menu.background": "#303031",
+		"menu.foreground": "#ffffff",
```

Comment:
The main foreground color feels a bit too "loud" in the dark theme, light theme looks great. The only other place where we have a brighter white is in the settings title and that is using `#e7e7e7`, not sure if you've tried that yet?

![image](https://user-images.githubusercontent.com/35271042/167766377-60f5d27f-c20c-4d20-b8e1-2268d433c309.png)